src/Symfony/Component/Translation/TranslatorBag.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+use Symfony\Component\Translation\Loader\LoaderInterface;
+
+/**
+ * TranslatorBag loads catalogue from resources.
+ *
+ * @author Abdellatif Ait boudad <a.aitboudad@gmail.com>
+ */
+class TranslatorBag implements TranslatorBagInterface, TranslatorBagIdentifiable
```

Comment:
Others `TranslatorBag` have a named (Cache, Fallback) except this one.
I would have renamed it to `ResourceTranslatorBag`.